images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.4
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
+        -c cpg -c bioconda -c conda-forge \
+        r-essentials r-base=4.1.2 python=3.10 \
+        anaconda cmake \
+        gettext lapack r-matrix \
+        r r-rcpp r-rcpparmadillo r-data.table r-bh r-matrix \
+        r-spatest r-rcppeigen r-devtools r-skat \
+        r-rcppparallel r-optparse boost openblas r-rhpcblasctl r-metaskat \
+        r-skat r-qlcmatrix r-rsqlite && \
+    rm -r /root/micromamba/pkgs && \
+    pip3 install cget click
```

Comment:
Do you need `cget`?

Reply:
again I assume she wouldn't have [listed it](https://github.com/saigegit/SAIGE/blob/main/conda_env/createCondaEnvSAIGE_steps.txt) unless it wasn't necessary, but I guess I could try running the actual package with a reduced set of dependencies and see?

Reply:
It's not a huge deal, I was just surprised to see it listed. I'd be surprised if it was actually necessary.

Reply:
ok! It's building again now, will see if it works.
I just realised that this is actually still only installing all of the dependencies, not the package itself? Can I do that from source (from [here](https://saigegit.github.io/SAIGE-doc/docs/Installation_conda.html)) within the Dockerfile?:

>   src_branch=main
    repo_src_url=https://github.com/saigegit/SAIGE
    git clone --depth 1 -b $src_branch $repo_src_url
    Rscript ./SAIGE/extdata/install_packages.R
    R CMD INSTALL --library=path_to_final_SAIGE_library SAIGE

Reply:
Yeah, it's fairly annoying they neither publish a package nor version their releases. So we can't even easily build this reproducibly without hardcoding a GitHub SHA here...

I'll change this slightly and also integrate it with the top-level versioning of this repo.

Reply:
ok! To test, would we be able to just run, e.g.,:

```Rscript createSparseGRM.R --help```

to make sure the various functions / scripts have been installed properly?